src/Support/MediaStream.php

```diff
@@ -98,39 +98,50 @@ public function getZipStream(): ZipStream
     protected function getZipStreamContents(): Collection
     {
         return $this->mediaItems->map(fn (Media $media, $mediaItemIndex) => [
-            'fileNameInZip' => $this->getZipFileNamePrefix($this->mediaItems, $mediaItemIndex).$this->getFileNameWithSuffix($this->mediaItems, $mediaItemIndex),
-            'media' => $media,
-        ]);
+        'fileNameInZip' => $this->getZipFileNamePrefix($this->mediaItems, $mediaItemIndex) . $this->getFileNameWithSuffix($this->mediaItems, $mediaItemIndex),
```

Comment:
Could you make sure that there are no code style changes in this PR. Unnecessary changes like his make the PR harder to review.